internal/provider/data_source_aci_application_epg_test.go

```diff
@@ -14,21 +14,19 @@ import (
 func TestAccDataSourceFvAEPgWithFvAp(t *testing.T) {
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { testAccPreCheck(t) },
+		PreCheck:                 func() { testAccPreCheck(t, "apic", "1.0(1e)-") },
 		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
 		Steps: []resource.TestStep{
 			{
 				Config:             testConfigFvAEPgDataSourceDependencyWithFvAp,
 				ExpectNonEmptyPlan: false,
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckResourceAttr("data.aci_application_epg.test", "name", "test_name"),
-					resource.TestCheckResourceAttr("data.aci_application_epg.test", "admin_state", "no"),
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.